script.js

```diff
@@ -170,3 +171,24 @@ form.addEventListener('submit', (event) => {
     event.preventDefault();
   }
 });
+
+// STORE VALUES LOCALLY
+
+const inputText = document.querySelectorAll('.input-text');
+const dataStored = {
+  name: '',
+  email: '',
+  comments: '',
+};
+inputText.forEach((input) => {
+  input.addEventListener('input', () => {
+    dataStored[input.name] = input.value;
+    localStorage.setItem('everyData', JSON.stringify(dataStored));
+  });
+});
+const formStored = JSON.parse(localStorage.getItem('everyData'));
+if (formStored) {
+  inputText.forEach((element) => {
+    element.value = formStored[element.name];
+  });
```

Comment:
- [ ] when I write now in the textarea and reload the page I lost the name and the email please make sure to fix that also when I update the name I lost the other please try to tests all the possibilities

Reply:
Can I merge now?

Reply:
@RaphDasilva it's still there here a loom video to help you https://www.loom.com/share/e2fb9f115c174c618b7f1a82b7cc1c2d

Reply:
You have to submit it first... Before refreshing

Reply:
@youmari please watch my loom video to see what am saying .

https://www.loom.com/share/7d78bf6404e346dd95ad898c038b13a0